### PR TITLE
adding a line from @MDSCLLC - studying

### PR DESCRIPTION
**Pull request by MDSCLLC**
Adding a line from @MDSCLLC - studying

delgueta23_cartoon

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
 Hello there
-Lets try SSH
\ No newline at end of file
+Lets try SSH
+
+This line belongs to David (@MDSLLC): Studying AI/ML (Google)
```
Comment (MDSCLLC) [inline test.txt]: How is @MDSLLC?

**Review by MDSCLLC — COMMENTED** [review]
oopsies, @MDSCLLC

File: test.txt (hunk 1 above)
Comment (nbkhope) [reply test.txt]: ok!

File: test.txt [hunk 2]
```diff
@@ -1,4 +1,7 @@
 Hello there
 Lets try SSH
 
+<<<<<<< HEAD
```
Comment (nbkhope) [inline test.txt]: Could you remove these conflict lines? `<<<<<<< HEAD`

Also in the bottom `>>>>>>> upstream/master`

**Review by nbkhope — APPROVED** [review]

**Comment by nbkhope** [comment]
Looks good to me (LGTM) 🚀

**Pull request merged**